extensions/resource-deployment/src/ui/eulaWizard/eulaWizard.ts

```diff
@@ -0,0 +1,153 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+import { INotebookService, Notebook } from '../../services/notebookService';
+import { IToolsService } from '../../services/toolsService';
+import { Model } from '../model';
+import { InputComponents, setModelValues } from '../modelViewUtils';
+import { WizardBase } from '../wizardBase';
+import { DeploymentProvider, DeploymentType, EulaWizardDeploymentProvider, instanceOfEulaWizardDeploymentProvider, NotebookWizardInfo, ResourceType } from '../../interfaces';
+import { IPlatformService } from '../../services/platformService';
+import { NotebookWizardAutoSummaryPage } from './eulaWizardAutoSummaryPage';
+import { EulaWizardPage } from './eulaWizardPage';
+import { EulaInformation, EulaSettingsPage } from './eulaSettingsPage';
+//import { EulaSettingsPage } from './eulaSettingsPage';
+
+const localize = nls.loadMessageBundle();
+
+export class EulaWizard extends WizardBase<EulaWizard, EulaWizardPage, Model> {
```

Comment:
EulaWizard has so much functionality of NotebookWizard that it would be better and cleaner to make extend NotebookdWizard

Reply:
Eventually, this is going to replace the Notebook Wizard. That is why I did not spend time in extending it.

Reply:
I have reservations on that approach. Replace a class that is itself evolving in this manner is a risky venture. IMO, just creating a EulaWizardPage and sliding that into all wizards is the easier thing to do.